index.css

```diff
@@ -1,5 +1,6 @@
 body {
   font-family: 'Lato', sans-serif;
+  overflow-x: hidden;
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/219131687-da039266-369e-462a-81aa-5ee967d8806b.png)

- [ ] As the previous reviewer suggested that your mobile design :thinking: is scrollable to horizontal on the about page which is not professional at all. Kindly fix it :+1: to make your project professional and to follow the design guidelines :+1: